codespell_lib/data/dictionary.txt

```diff
@@ -45762,7 +45874,12 @@ repostiories->repositories
 repostiory->repository
 repostories->repositories
 repostory->repository
-repote->report, remote,
+repote->report, remote, repute,
```

Comment:
Also `repose`? If so, apply to all verbal forms.

Reply:
I couldn't find this typo.